docs/cassandra/multi-az.md

```diff
@@ -0,0 +1,51 @@
+Cassandra across multiple availability zones
+============================================
+
+Navigator supports running Cassandra with rack and datacenter-aware
+replication. To deploy this, you must run a `nodePool` in each availability
+zone, and mark each as a separate Cassandra rack.
+
+The `nodeSelector` field of a nodePool allows scheduling the nodePool to a set
+of nodes matching labels. This should be used with a node label such as
+`failure-domain.beta.kubernetes.io/zone`.
+
+The `datacenter` and `rack` fields mark all Cassandra nodes in a nodepool as
+being located in that datacenter and rack. This information can then be used
+with the `NetworkTopologyStrategy` keyspace replica placement strategy.
+
+As an example, the nodePool section of a CassandraCluster spec for deploying
+into GKE in europe-west1:
+
+```yaml
+  nodePools:
+  - name: "np-europe-west1-b"
+    replicas: 3
+    datacenter: "europe-west1"
+    rack: "europe-west1-b"
```

Comment:
:grey_question: This makes me wonder what happens if I used `rack: "b"`. Does Cassandra know that rack b in europe-west1 is different than rack b in us-west1?

Reply:
And if not, perhaps we should add a note that rack labels must be unique.

Reply:
Ec2Snitch simply labels a rack in `us-east-1a` as `1a`, so it seems cassandra knows the difference:

https://github.com/apache/cassandra/blob/4a50f4467f439f9a00aac6aa4d3b83a6c1c8d6c4/src/java/org/apache/cassandra/locator/Ec2Snitch.java#L54-L62